src/client/components/map-vote-dialog/dialog.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { PureComponent } from 'react';
+import injectSheet from 'react-jss';
+import Aux from 'react-aux';
+import {
+  Button,
+  RadioButton,
+  RadioButtonGroup,
+  Typography,
+} from 'materialize-react';
+import PropTypes from 'prop-types';
+
+/**
+ * The map voting dialog component.
+ *
+ * @class
+ */
+class DialogComponent extends PureComponent {
+  static propTypes = {
+    classes: PropTypes.shape({ container: PropTypes.shape({}) }).isRequired,
+    maps: PropTypes.arrayOf(PropTypes.string).isRequired,
+    onCanceled: PropTypes.func.isRequired,
+    onAccepted: PropTypes.func.isRequired,
+  };
+
+  static styles = { container: { padding: '10px' } }
+
+  /**
+   * Handles the 'OK' button pressed.
+   */
+  handleAccepted = () => {
+    const selectedMap = this.radioButtonGroup.selected;
+
+    this.props.onAccepted(selectedMap);
+  }
+
+  render() {
+    const {
+      maps,
+      onCanceled,
+    } = this.props;
+
+    return (
+      <Aux>
+        <div className={this.props.classes.container}>
+          <Typography typography="headline">Vote for map</Typography>
+        </div>
+        <div>
+          <RadioButtonGroup
```

Comment:
Check out how the new RadioButtons work: https://github.com/TF2PickupNET/TF2Pickup/blob/dev/src/client/components/post-user-creation-dialog/region-section.jsx#L85